### ISSUE TITLE
SVG Command (a,A)

### ISSUE THREAD

**Issue by junaid04** [timrwood/SVGPath#14]
This library is great and helpful for create path from svg. But it does not support (a,A) command of svg.

**Comment by ReflectCode** [comment]
Yes support for (a,A) command will help cover all commands in SVG paths.
